README.md

```diff
@@ -165,10 +177,17 @@ List of podcasts which are helpful for software engineers/programmers.
   - <b>Frequency</b>  : Once every week
 
 ## Kotlin
+
 - [Talking kotlin](http://talkingkotlin.com)
 
   - <b>Description</b>: Kotlin and more
   - <b>Frequency</b>  : Approximately once a month
+  
+## Swift
+
+- Swift Unwrapped ([By Spec](https://spec.fm/podcasts/swift-unwrapped) | [iTunes](https://itunes.apple.com/us/podcast/swift-unwrapped/id1209817203?mt=2)
```

Comment:
Can we fix the closing angular bracket here, plz check the preview

Reply:
edit this Thanks!